Descent3/init.cpp

```diff
@@ -1988,8 +1988,14 @@ void SetupTempDirectory(void) {
   if (t_arg) {
     strcpy(Descent3_temp_directory, GameArgs[t_arg + 1]);
   } else {
-    // initialize it to custom/cache
-    ddio_MakePath(Descent3_temp_directory, Base_directory, "custom", "cache", NULL);
+    ddio_MakePath(Descent3_temp_directory, std::filesystem::temp_directory_path().u8string().c_str(), "Descent3",
```

Comment:
Need to use non-throwable version of `temp_directory_path()` and check ec error on path construction.

Reply:
fixed